test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -473,6 +474,11 @@ describe("GET tests", () => {
 
   test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with trust data and feature flag off`, async () => {
     mockIsActiveFeature.mockReturnValue(false); // FEATURE_FLAG_ENABLE_TRUSTS_WEB flag
+    mockIsActiveFeature.mockReturnValueOnce(false);
```

Comment:
Are these all new calls to get the `FEATURE_FLAG_ENABLE_REDIS_REMOVAL` value? If yes, would be helpful to add as comment, like elsewhere.

Reply:
Comment added.